case_utils/case_validate/__init__.py

```diff
@@ -118,15 +118,44 @@ def main() -> None:
       help="(ALMOST as with pyshacl CLI) Send output to a file.  If absent, output will be written to stdout.  Difference: If specified, file is expected not to exist.  Clarification: Does NOT influence --format flag's default value of \"human\".  (I.e., any machine-readable serialization format must be specified with --format.)",
       default=sys.stdout,
     )
+    parser.add_argument(
+      '-r',
+      '--recursive',
+      action='store_true',
+      default=False,
+      help="Allows passing of a directory for recursive validation of multiple CASE files in one execution",
+    )
+    parser.add_argument(
```

Comment:
The use case was if you had an automated process that was generating CASE files amongst other artifacts but only wanted to validate the `.json` files or similar. But it's probably a limited enough use case that it can get yanked.